R/pretty_lollipop_plot.R

```diff
@@ -2,58 +2,118 @@
 #'
 #' @description Generates a visually appealing and interactive lollipop plot.
 #'
-#' @details This function is depending on a modified version of `readMAF` from the `g3viz` package.
-#' Returned plot is interactive, meaning the user can hover over individual points in the plot to reveal more information.
-#' The plot can also be exported in a variety of file formats inside the interactive view of the lollipop plot.
+#' @details Retrieve a maf of a specific sample or a set of samples. Returned plot is interactive, meaning the 
+#' user can hover over individual points in the plot to reveal more information. The plot can also be exported 
+#' in a variety of file formats inside the interactive view of the lollipop plot.
 #'
-#' @param maf_df A data frame containing the mutation data (from a MAF).
-#' @param gene The gene symbol to plot.
-#' @param plot_title Optional (defaults to gene name).
-#' @param plot_theme Options: cbioportal(default), blue, simple, nature, nature2, ggplot2, and dark.
-#' @param out_name Optional, set the file name of the plot, if you export it to disk. Default name is my_lollipop_plot_{gene}.
-#'
-#' @return Nothing.
+#' @param maf_data A data frame containing the mutation data (from a MAF).
+#' @param gene_symbol The gene symbol to plot.
+#' 
+#' @return A lolipop plot.
 #'
-#' @import g3viz dplyr
+#' @import tidyverse BiocManager maftools
```

Comment:
tidyverse package cannot be specified as dependency here because it technically cannot be imported. Instead, individual packages need to be specified. For example, `dplyr`. Feel free to add other dependencies as needed.
Another note here is about the other 2 dependencies - why do we need BiocManager and maftools here?

Reply:
whoops! 

maftools is needed for the read.maf() line 34?